src/AzureIoTHub.Portal/Client/Pages/Devices/DeviceDetailPage.razor

```diff
@@ -27,10 +27,10 @@
     <MudGrid>
         <MudItem xs="12" sm="4" md="3">
             <MudCard>
-                <MudCardHeader>
+                <MudCardHeader Class="DeviceCardHeader">
```

Comment:
Please also think about updating DeviceCreation page, IoT Edge creation and detail page